compile.js

```diff
@@ -9,31 +9,37 @@
   process.CLIENT_COMPILE = true;
 
   var habitat = require("habitat"),
-      env = habitat.load(),
-      flickrOptions = env.get("FLICKR"),
-      APIBuilder = require("./src/flickr-api-object.js"),
-      Utils = require("./src/utils.js"),
-      filename = "flickrapi";
-
-  if(process.argv.indexOf("dev") > -1) {
+    env = habitat.load(),
```

Comment:
can you please restore the indentation that was used previously, so that the vars line up?